### ISSUE TITLE
Past Performance Questionnaire (Section 10., Page 15 of RFQ)

### ISSUE THREAD

**Issue by MorphoTrust01** [18F/tts-buy-document-authentication-services#10]
Would GSA consider allowing Past Performance references and PPQs on behalf of subcontractors, with the Small Business as the prime contractor?  Many of the specified/requested capabilities are highly specialized to vendors working in the document authentication space.  Many instances of past performance GSA seeks may be provided by non-Small Business technology providers versus the Small Business prime contractor.

**Comment by oghaffari** [comment]
Yes. PPQs are intended to be from all partners involved in the bid.
